src/core/border_router/routing_manager.cpp

```diff
@@ -3683,12 +3700,29 @@ RoutingManager::PdPrefixManager::State RoutingManager::PdPrefixManager::GetState
 
     if (mEnabled)
     {
-        state = mIsRunning ? kDhcp6PdStateRunning : kDhcp6PdStateStopped;
+        state = mIsStarted ? (mIsPaused ? kDhcp6PdStateIdle : kDhcp6PdStateRunning) : kDhcp6PdStateStopped;
     }
 
     return state;
 }
 
+void RoutingManager::PdPrefixManager::Evaluate(void)
+{
+    const FavoredOmrPrefix &favoredPrefix = Get<RoutingManager>().mOmrPrefixManager.GetFavoredPrefix();
+
+    if (favoredPrefix.IsEmpty() || favoredPrefix.GetPreference() < OmrPrefixManager::kPdRoutePreference ||
+        favoredPrefix.GetPrefix() == mPrefix.GetPrefix())
+    {
+        LogInfo("Trying to request prefix via DHCPv6 PD");
```

Comment:
Suggest removing this logs here and in the following `else` block. They may be logged frequently, even when the PD prefix state remains unchanged, causing potential confusion.

`Evaluate()` is called often, and not every call results in a state change in `PdPrefixmanager`. With the current code, we would log either `"Trying to request prefix via DHCPv6 PD"` or `"Stop requesting prefix via DHCPv6 PD"` on each call.

We already log the state changes in `EvaluateStateChange()` (only when there is an actual state change). This should be sufficient to track the relevant events without generating extra log messages.

Reply:
Done